tests/west/runner/test_nrfjprog.py

```diff
@@ -0,0 +1,239 @@
+     ['nrfjprog', '--reset', '-f', 'NRF51', '--snr', TEST_OVR_SNR]],
+
+    '1YYY':
+    [['nrfjprog', '--eraseall', '-f', 'NRF51', '--snr', TEST_OVR_SNR],
+     ['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF51', '--snr', TEST_OVR_SNR],  # noqa: E501
+     ['nrfjprog', '--reset', '-f', 'NRF51', '--snr', TEST_OVR_SNR]],
+
+    # NRF52:
+    '2NNN':
+    [['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_DEF_SNR, '--sectoranduicrerase'],  # noqa: E501
+     ['nrfjprog', '--pinresetenable', '-f', 'NRF52', '--snr', TEST_DEF_SNR],
+     ['nrfjprog', '--pinreset', '-f', 'NRF52', '--snr', TEST_DEF_SNR]],
+
+    '2NNY':
+    [['nrfjprog', '--eraseall', '-f', 'NRF52', '--snr', TEST_DEF_SNR],
+     ['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_DEF_SNR],  # noqa: E501
+     ['nrfjprog', '--pinresetenable', '-f', 'NRF52', '--snr', TEST_DEF_SNR],
+     ['nrfjprog', '--pinreset', '-f', 'NRF52', '--snr', TEST_DEF_SNR]],
+
+    '2NYN':
+    [['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_OVR_SNR, '--sectoranduicrerase'],  # noqa: E501
+     ['nrfjprog', '--pinresetenable', '-f', 'NRF52', '--snr', TEST_OVR_SNR],
+     ['nrfjprog', '--pinreset', '-f', 'NRF52', '--snr', TEST_OVR_SNR]],
+
+    '2NYY':
+    [['nrfjprog', '--eraseall', '-f', 'NRF52', '--snr', TEST_OVR_SNR],
+     ['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_OVR_SNR],  # noqa: E501
+     ['nrfjprog', '--pinresetenable', '-f', 'NRF52', '--snr', TEST_OVR_SNR],
+     ['nrfjprog', '--pinreset', '-f', 'NRF52', '--snr', TEST_OVR_SNR]],
+
+    '2YNN':
+    [['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_DEF_SNR, '--sectoranduicrerase'],  # noqa: E501
+     ['nrfjprog', '--reset', '-f', 'NRF52', '--snr', TEST_DEF_SNR]],
+
+    '2YNY':
+    [['nrfjprog', '--eraseall', '-f', 'NRF52', '--snr', TEST_DEF_SNR],
+     ['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_DEF_SNR],  # noqa: E501
+     ['nrfjprog', '--reset', '-f', 'NRF52', '--snr', TEST_DEF_SNR]],
+
+    '2YYN':
+    [['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_OVR_SNR, '--sectoranduicrerase'],  # noqa: E501
+     ['nrfjprog', '--reset', '-f', 'NRF52', '--snr', TEST_OVR_SNR]],
+
+    '2YYY':
+    [['nrfjprog', '--eraseall', '-f', 'NRF52', '--snr', TEST_OVR_SNR],
+     ['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF52', '--snr', TEST_OVR_SNR],  # noqa: E501
+     ['nrfjprog', '--reset', '-f', 'NRF52', '--snr', TEST_OVR_SNR]],
+}
+
+
+#
+# Fixtures
+#
+
+@pytest.fixture
+def nrfjprog_init():
+    '''NrfJprogBinaryRunner constructed with __init__ args/kwargs.
+
+    Returns a factory that consumes a runner configuration and
+    nrfjprog runner __init__ arguments.'''
+    def _factory(cfg, family, softreset, snr, erase):
+        args = [cfg, family, softreset, snr]
+        return NrfJprogBinaryRunner(*args, erase=erase)
```

Comment:
Could this be just `return NrfJprogBinaryRunner(cfg, family, softreset, snr, erase=erase)`?